sass/includes/search/_search-filters.scss

```diff
@@ -4,7 +4,8 @@
     padding: 1rem;
 
     &__heading {
-        font-size: 1.3em;
+        // font-size: 1.3em;
+        display: none;
```

Comment:
Do we still want this to be available for screen readers?

Reply:
I'm turning this off visually for now as these headings will be part of the accordion piece and will be restored when that gets integrated.